pricing-app/src/app/order/OrderCoffeeItem.tsx

```diff
@@ -0,0 +1,168 @@
+import { BreakFast, DrinkType, Milk, Size, Topping } from "@/types/enum";
+import { FC, useState } from "react";
+import SelectionBox from "@/components/selectionBox";
+import Options from "@/types";
+import { defaultCoffee } from "@/mock/data";
+import { Item } from "@/types/type";
+import { v4 as uuid } from 'uuid';
+
+type Props = {
+  type: DrinkType | BreakFast;
+  toppingOptions: Options[];
+  sizeOptions: Options[];
+  milkOptions: Options[];
+  onCoffeeItemAdded: (coffee: Item) => void;
+};
+
+const OrderCoffeeItem: FC<Props> = ({
+  type,
+  toppingOptions,
+  sizeOptions,
+  milkOptions,
+  onCoffeeItemAdded,
+}: Props) => {
+  const calculatePrice3 = (coffee: Item): number => {
```

Comment:
Should separated this out. Ideally, each component should tell a simple, straightforward story.